documentation/src/main/asciidoc/ch09.asciidoc

```diff
@@ -445,21 +452,22 @@ include::{sourcedir}/org/hibernate/validator/referenceguide/chapter09/OSGiScript
 ----
 ====
 
-It is designed specifically for OSGi environments and allows user to pass `BundleContext` as a
+It is designed specifically for OSGi environments and allows you to pass the `BundleContext` as a
 parameter which will be used to search for `ScriptEngineFactory`.
 
-As it was already said, it is also possible to use other than https://www.jcp.org/en/jsr/detail?id=223[JSR-223]
-based script evaluation (e.g. https://docs.spring.io/spring/docs/current/spring-framework-reference/core.html#expressions[Spring Expression Language]).
-`SpringELEvaluatorFactory` can be written like:
+As already mentioned, you can also use script engines that are not based on https://www.jcp.org/en/jsr/detail?id=223[JSR 223].
+
+For instance, to use the https://docs.spring.io/spring/docs/current/spring-framework-reference/core.html#expressions[Spring Expression Language]),
+you can define a `SpringELEvaluatorFactory` as:
```

Comment:
Probably `SpringELEvaluatorFactory` -> `SpringELScriptEvaluatorFactory` as the name changed

Reply:
Good catch, fixed.